jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -49,8 +57,60 @@
 /*[ENDIF]*/
 			PlatformLoggingMXBean {
 
-	private static final LoggingMXBeanImpl instance = new LoggingMXBeanImpl();
+	private static final LoggingMXBeanImpl instance;
+
+/*[IF Sidecar19-SE]
+	private static final Module java_logging;
```

Comment:
Does this code need to be commented out? As much as possible the code should not be in comments to help avoid missing something when the code is changed. This applies to the initialization code as well.

Reply:
@keithc-ca you probably have an opinion on this since you try to keep your workspace clean

Reply:
Okay thanks. I've seen the directives commenting done a few ways so I wasn't sure what is preferred.

Reply:
My preference is that they are only single line comments except in cases where a Java-aware editor would be confused (e.g. two imports or declarations of the same name).

Reply:
My guidelines are to comment out only when the code cannot compile. The master code isn't expected to work. If something needs to be commented, keep the latest code (i,e, Java 9) uncommented.